app/main.py

```diff
@@ -1,3 +1,56 @@
+from __future__ import annotations
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
```

Comment:
you use `float` here, and `int` in other methods. Fix it please, km could be either `int` or `float`